misc/extension_api_validation/4.4-stable.expected

```diff
@@ -328,3 +328,10 @@ GH-108825
 Validate extension JSON: Error: Field 'classes/EditorExportPlatformExtension/methods/_get_option_icon/return_value': type changed value in new API, from "ImageTexture" to "Texture2D".
 
 Return type changed to allow returning both ImageTexture and DPITexture. Compatibility method registered.
+
+
+GH-110250
+---------
+Validate extension JSON: Error: Hash changed for 'classes/Control/methods/has_focus', from 0232A5E1 to 859196D4. This means that the function has changed and no compatibility function was provided.
```

Comment:
This should be the error printed about the number of arguments, for each method, not the hash itself

Reply:
:melting_face: